.circleci/config.yml

```diff
@@ -224,13 +224,10 @@ jobs:
           steps:
             - run:
                 name: Move Xcode project aside
-                command: mv MapboxNavigation.xcodeproj MapboxNavigation.xcodeproj-aside
+                command: rm -rf *.xcodeproj
             - run:
                 name: MapboxNavigation-Package
                 command: xcodebuild -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=<< parameters.iOS >>,name=<< parameters.device >>' -scheme MapboxNavigation-Package build <<# parameters.test >>test <</ parameters.test >> <<# parameters.codecoverage >>-enableCodeCoverage YES<</ parameters.codecoverage >>
-            - run:
-                name: Move Xcode project back in place
-                command: mv MapboxNavigation.xcodeproj-aside MapboxNavigation.xcodeproj
       - when:
           condition: << parameters.codecoverage >>
           steps:
```

Comment:
Interesting, I though we don't use codecov anymore

Reply:
We don't use. It is always false as far as I see.